google-finance/src/components/Nav/NavCategory.jsx

```diff
@@ -13,11 +22,18 @@ const NavCategory = ({ icon, text }) => {
 const StyledSpan = styled.span`
   display: flex;
   align-items: center;
-
+  width: 19.8rem;
+  height: 4.4rem;
   gap: 1.4rem;
-  margin-bottom: 3rem;
-
+  position: relative;
+  left: -1.5rem;
+  margin-bottom: 2rem;
+  border-radius: 0rem 2.2rem 2.2rem 0rem;
+  background-color: ${(props) => (props.isActive ? props.theme.colors.blue_background : 'transparent')};
```

Comment:
```suggestion
&:active {
    background-color: props.theme.colors.blue_background ;
  }
````
가영이가 한 방법도 좋닷!! 훔 다른 방법을 생각해보자면 이런식으로도 가능할 것 같아용!

Reply:
앗 위에 예시코드에 theme 넣어서 해야대는데 그거 빼먹었다 무튼 ㅎ.ㅎ